pkg/apis/stork/register.go

```diff
@@ -2,7 +2,7 @@ package stork
 
 const (
 	// GroupName is the group name of the CRD
-	GroupName = "stork.com"
```

Comment:
GroupName should be in domain format, stork.libopenstorage.com

Reply:
Updated. It's a ghost link and has only symbolic purpose.

Reply:
It should actually be used to register the CRD. The actual group name used to generate the boilerplate code in doc.go should also be stork.libopenstorage.com

Reply:
fixed the group name